.github/workflows/ci-test-xtgeo.yml

```diff
@@ -1,61 +1,15 @@
+# build and test some end points
 name: buildsall
 
 on:
   pull_request:
     branches: [master]
 
-env:
-  CIBW_BEFORE_ALL_LINUX: "sh scripts/swig_manylinux.sh"
-  CIBW_BEFORE_ALL_WINDOWS: "choco install -y --no-progress --allow-empty-checksums -r swig --version=4.0.1"
-  CIBW_BEFORE_ALL_MACOS: "brew install swig"
-  CIBW_BEFORE_TEST: |
-    cd {project}
-    pip install -r requirements/requirements.txt
-    pip install -r requirements/requirements_test.txt
-
-  CIBW_TEST_COMMAND: |
-    cd {project}
-    git clone --depth 1 https://github.com/equinor/xtgeo-testdata ../xtgeo-testdata
-    pytest tests --disable-warnings
-
-  CIBW_BUILD: |
-    cp39-manylinux_x86_64
-    cp38-macosx_x86_64
-    cp38-win_amd64
-
-# Verify that wheel build jobs succeed
 jobs:
-  build_multiwheels:
-    name: CIBW PY38 or 39 on ${{ matrix.os }}
```

Comment:
Now building with `cibuildwheel` will only be performed when tag is pushed and the `deploy-xtgeo-pypi.yml ` is run?